jekyll/_components/tables.md

```diff
@@ -2,17 +2,142 @@
 layout: page
 title: Tables
 order: 9
-draft: true
```

Comment:
I think we might want to leave this as a draft until it's merged into the main Design System (also because there are probably some table usages in the non-DevTools parts of Firefox that we'll want to mention).